contexts/webkms-v1.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "@protected": true,
+
+    "AesKeyWrappingKey2019": "https://w3id.org/security#AesKeyWrappingKey2019",
+    "DeleteKeyOperation": "https://w3id.org/security#DeleteKeyOperation",
+    "DeriveSecretOperation": "https://w3id.org/security#DeriveSecretOperation",
+    "ExportKeyOperation": "https://w3id.org/security#ExportKeyOperation",
+    "GenerateKeyOperation": "https://w3id.org/security#GenerateKeyOperation",
+    "KmsOperation": "https://w3id.org/security#KmsOperation",
+    "RevokeKeyOperation": "https://w3id.org/security#RevokeKeyOperation",
+    "Sha256HmacKey2019": "https://w3id.org/security#Sha256HmacKey2019",
```

Comment:
I think we want a separate context for this key type... or any key type.

Reply:
@dlongley k, extracted to https://github.com/digitalbazaar/sha256-hmac-key-2019-context/pull/1